lms/djangoapps/grades/models.py

```diff
@@ -138,14 +141,15 @@ class VisibleBlocks(models.Model):
     blocks_json = models.TextField()
     hashed = models.CharField(max_length=100, unique=True)
     course_id = CourseKeyField(blank=False, max_length=255, db_index=True)
+    version = models.IntegerField()
```

Comment:
I'm replacing the course_id index to use (version, course_id), but leaving the hashed index alone, since that's used for an fk relationship.